testing/scenario_app/ios/Scenarios/Scenarios/TextPlatformView.m

```diff
@@ -47,6 +47,7 @@ - (instancetype)initWithFrame:(CGRect)frame
     _textView = [[UITextView alloc] initWithFrame:CGRectMake(50.0, 50.0, 250.0, 100.0)];
     _textView.textColor = UIColor.blueColor;
     _textView.backgroundColor = UIColor.lightGrayColor;
+    _textView.userInteractionEnabled = YES;
```

Comment:
Can you provide some context on this?

Reply:
removed. This was a code added for an experiment that I try to use scenario app to test this.